sheetScript.gs

```diff
@@ -87,99 +94,123 @@ function updatePlaylists(sheet) {
       sheet.getRange(iRow + 1, reservedColumnTimestamp + 1).setValue(isodate);
       lastTimestamp = isodate;
     }
-  
+
     // Check if it's time to update already
-    var freqDate = new Date(lastTimestamp);
-    var dateDiff = Date.now() - freqDate;
-    var nextTime = data[iRow][reservedColumnFrequency]  * MILLIS_PER_HOUR;
-    if (nextTime && dateDiff <= nextTime) {
+    var lastDate = new Date(lastTimestamp);
+    var lastTime = lastDate.getTime();
+    var dateDiff = Date.now() - lastTime;
```

Comment:
Instead of this, we can just subtract two date objects, aka `new Date() - lastDate`, and we don't need to create lastTime